azerothcore/mod-autobalance
# Feat: In-game hints support internationalized text.

## Changes Proposed:
I have added internationalized text support for client prompts in Korean, French, German, Simplified Chinese, Traditional Chinese, Spanish, and Russian for all prompts in the game.

## Issues Addressed:

- Closes

## SOURCE:


## Tests Performed:


I successfully compiled on Ubuntu 22.04 without any errors or warnings. Since it's just text internationalization, I verified using two characters with Simplified Chinese and English clients, and didn't find any issues.

The following are some version information and parameters during compilation:

- CMake version: 3.22.1
- Clang version: 14.0.0
-  AzerothCore revision: b6ee4f49cfbf 2024-02-13 19:21:13 +0000 (master branch)


## How to Test the Changes:


1. Upon entering the game, I selected a character and entered a random instance to check if the prompt messages have been internationalized.

2. Using GM privileges, I executed the .ab command to check if the prompts have been internationalized.
